pkgs/top-level/all-packages.nix

```diff
@@ -24550,6 +24550,11 @@ with pkgs;
 
   moq = callPackage ../development/tools/moq { };
 
+  wrapWithMoveProverDeps = callPackage ../development/tools/move/wrapWithMoveProverDeps.nix { };
```

Comment:
```suggestion
  move-wrapWithMoveProverDeps = callPackage ../development/tools/move/wrapWithMoveProverDeps.nix { };
```

Reply:
Other CLIs should depend on this (eg Aptos/Sui)

Reply:
What do the programs have in common? is it maybe related to some framework?